deploy/example/azschedulerextender/csi-azuredisk-azschedulerextender.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: v1
```

Comment:
I am most certain that in case of aks and aks-engine we will not deploy the extender this way. I have addressed this in detail in the other comment.
Let me know and I can move this to deploy if we think that's more appropriate for now.

Reply:
I'm fine with the current location since we haven't yet figured out the deployment plan for the scheduler extender. We can finalize the location later.